### DISCUSSION TITLE
[Ente Photos] Allow option to kill app on close - Desktop

### DISCUSSION THREAD

**Discussion by zoom6453** [ente-io/ente#3616] [Enhancements]
When the app gets closed, the app is only minimised, and I have to manually right click on the tray icon and select 'Quit Ente'.

There should be an option to choose what closing the app does.

I'm on linux mint 22

**Comment by Ronelzz** [comment]
I'd like the same on windows please. It's been implemented in Ente Auth, so shouldn't be that hard to do the same for Photos.
